sh/cachix

```diff
@@ -6,7 +6,7 @@ fail () {
 }
 
 if [ -z "$CACHIX_SIGNING_KEY" ]
-then fail "The CACHIX_AUTH_TOKEN environment variable needs to be set."
```

Comment:
I believe the `cachix authtoken` and `cachix use` commands are still necessary?

Reply:
They're still in this file. I've removed the `authtoken` command from `.travis.yml`, but kept `use`.